components/library/tntnet/tntnet-server.p5m

```diff
@@ -0,0 +1,50 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Jim Klimov
+#
+
+<transform file path=usr.*/man/.+ -> default mangler.man.stability uncommitted>
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)/server@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY) - appserver"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file path=usr/bin/$(MACH64)/tntnet
+file path=usr/bin/tntnet
+
+file path=usr/share/man/man8/tntnet.8
+file path=usr/share/man/man7/tntnet.properties.7
+file path=usr/share/man/man7/tntnet.xml.7
+
+file path=usr/share/tntnet/tntnet-conf2xml.pl
+
+dir path=var/log/tntnet group=webservd owner=webservd mode=0700
+dir path=var/www group=webservd owner=webservd mode=0700 preserve=true
+
+# We deliver an example appserver config, since some actual webapp
+# implementation and relevant config are expected from the end-user
+# or a package from some 3rd-party integrator. Default SMF manifest
+# depends on "/etc/tntnet/tntnet.xml" to be present, so by default it
+# would fail to start if enabled, and will not open up unconfigured
+# security breaches, not use resources until something reasonable
+# and meaningful is manually configured, etc.
+file Solaris/tntnet.xml.sample path=etc/tntnet/tntnet.xml.sample
```

Comment:
Please, move it out of /etc, for example to usr/share/tntnet/examples. It's annoying to have *.sample file in /etc, which even can't be deleted (IPS will recreate it on pkg fix)...